include/mxnet/lib_api.h

```diff
@@ -426,21 +432,30 @@ typedef void (*sparse_malloc_t)(void*, int, int, int, void**, int64_t**, int64_t
 
 #if defined(__NVCC__)
   typedef cudaStream_t mx_stream_t;
+  typedef curandStatePhilox4_32_10_t* mx_gpu_rand_pt;
```

Comment:
since this is a typedef, lets follow the naming style and call it mx_gpu_rand_t

should we typedef std::mt19937 to mx_cpu_rand_t too? or is that going too far?